editors/code/src/inlay_hints.ts

```diff
@@ -114,6 +124,17 @@ class HintsUpdater implements Disposable {
         this.syncCacheAndRenderHints();
     }
 
+    updateInlayHintsStyles() {
```

Comment:
Now we change the style unconditionally on every settings change, even if that setting is not related to hints per se.

This causes the hints to blink when changing some totally unrelated setting which does not look good.